src/current/v24.3/take-and-restore-locality-aware-backups.md

```diff
@@ -130,7 +130,7 @@ To restore from a specific backup, use [`RESTORE FROM {subdirectory} IN ...`]({%
 
 ## Create an incremental locality-aware backup
 
-If you backup to a destination already containing a [full backup]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#full-backups), an [incremental backup]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#incremental-backups) will be appended to the full backup in a subdirectory. When you're taking an incremental backup, you must ensure that the incremental backup localities match the full backup localities otherwise you will receive an error. Alternatively, take another full backup with the matching localities before running the incremental backup.
+If you back up to a destination already containing a [full backup]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#full-backups), an [incremental backup]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#incremental-backups) will be appended to the full backup in a subdirectory. To take a locality-aware incremental backup and restore from it successfully, you must maintain the same storage URI and `COCKROACH_LOCALITY` mappings that were established by the corresponding full backup. Alternatively, take another full backup with the matching localities and URIs before running the incremental backup.
```

Comment:
I think there was some confusion/ambiguity with my answer in Slack. The incremental mappings don't have to match the full backup mappings, they just have to be consistent. So if with one `BACKUP` command you have `incremental_location = ('foo?region=us-east', 'bar?region=us-west', 'baz?region=us-central')`, then that mapping must be maintained for all incrementals in that chain. @dt if you could confirm this behavior that'd be helpful!